{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"socketIsConnected"},{"kind":"text","text":": "},{"kind":"typeIdentifier","identifier":"doc:\/\/SwiftIO\/documentation\/SwiftIO\/Errno","preciseIdentifier":"s:7SwiftIO5ErrnoV","text":"Errno"},{"kind":"text","text":" { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["macOS"]}]},{"kind":"content","content":[{"anchor":"discussion","level":2,"type":"heading","text":"Discussion"},{"type":"paragraph","inlineContent":[{"type":"text","text":"A "},{"type":"codeVoice","code":"connect(2)"},{"type":"text","text":" or "},{"type":"codeVoice","code":"connectx(2)"},{"type":"text","text":" request was made"},{"type":"text","text":" "},{"type":"text","text":"on an already connected socket,"},{"type":"text","text":" "},{"type":"text","text":"or a "},{"type":"codeVoice","code":"sendto(2)"},{"type":"text","text":" or "},{"type":"codeVoice","code":"sendmsg(2)"},{"type":"text","text":" request was made"},{"type":"text","text":" "},{"type":"text","text":"on a connected socket specified a destination when already connected."}]},{"type":"paragraph","inlineContent":[{"type":"text","text":"The corresponding C error is "},{"type":"codeVoice","code":"EISCONN"},{"type":"text","text":"."}]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/swiftio\/errno\/socketisconnected"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/SwiftIO\/documentation\/SwiftIO\/Errno\/socketIsConnected","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Socket is already connected."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"socketIsConnected"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Errno","preciseIdentifier":"s:7SwiftIO5ErrnoV"}],"title":"socketIsConnected","roleHeading":"Type Property","role":"symbol","symbolKind":"property","externalID":"s:7SwiftIO5ErrnoV17socketIsConnectedACvpZ","modules":[{"name":"SwiftIO"}]},"hierarchy":{"paths":[["doc:\/\/SwiftIO\/documentation\/SwiftIO","doc:\/\/SwiftIO\/documentation\/SwiftIO\/Errno"]]},"references":{"doc://SwiftIO/documentation/SwiftIO/Errno/socketIsConnected":{"role":"symbol","title":"socketIsConnected","fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"socketIsConnected"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Errno","preciseIdentifier":"s:7SwiftIO5ErrnoV"}],"abstract":[{"type":"text","text":"Socket is already connected."}],"identifier":"doc:\/\/SwiftIO\/documentation\/SwiftIO\/Errno\/socketIsConnected","kind":"symbol","type":"topic","url":"\/documentation\/swiftio\/errno\/socketisconnected"},"doc://SwiftIO/documentation/SwiftIO/Errno":{"role":"symbol","title":"Errno","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"Errno"}],"abstract":[{"type":"text","text":"The Errno struct lists all the possible errors."}],"identifier":"doc:\/\/SwiftIO\/documentation\/SwiftIO\/Errno","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"Errno"}],"url":"\/documentation\/swiftio\/errno"},"doc://SwiftIO/documentation/SwiftIO":{"role":"collection","title":"SwiftIO","abstract":[{"type":"text","text":"The SwiftIO library allows you to access and control the hardware in an easy way."}],"identifier":"doc:\/\/SwiftIO\/documentation\/SwiftIO","kind":"symbol","type":"topic","url":"\/documentation\/swiftio"}}}